doc/rtd/tutorial/qemu.rst

```diff
@@ -80,6 +80,12 @@ server image using :command:`wget`:
 
     $ wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
 
+.. note::
+   Please be aware, the example is written for x86. It works on non-x86 but
+   is slower as it will need to be emulated. You can change the image type and
+   qemu-system-<arch> command name to match your architecture to make it again
```

Comment:
spelling issues on qemu-system not being seen, we need the following in both places:
```suggestion
   :spelling:ignore:`qemu-system-<arch>` command name to match your architecture to make it again
```